### DISCUSSION TITLE
[Bug]kafka组件@KafkaListener装饰器装饰的方法如果返回 true 执行 commit 会报错

### DISCUSSION THREAD

**Discussion by Yuliang-Lee** [midwayjs/midway#4042] [General / 闲聊]
问题一：
https://github.com/midwayjs/midway/blob/main/packages/kafka/src/framework.ts#L138 
这一行代码的传参有问题，会导致报错
<img width="1179" alt="image" src="https://github.com/user-attachments/assets/f0a08f0e-0daf-45f7-be9c-6e1822957199">

问题二：
这个返回 `true` 就会触发 `commit` 的逻辑在文档里也没说明，这里建议文档加上或者去掉这个 commit 的逻辑

**Comment by Yuliang-Lee** [comment]
https://github.com/midwayjs/midway/pull/4043

**Comment by waitingsong** [comment]
https://github.com/midwayjs/midway/pull/4154
